templates/admin/audit_log/index.html.ep

```diff
@@ -11,7 +11,11 @@
     %= include 'layouts/admin_nav'
 
     <div class="col-sm-10">
-        <h2><%= title %></h2>
+        <h2><%= title %>
+            % if (!$audit_enabled) {
+                - <span style="color: darkred;">auditing disabled</span>
+            % }
```

Comment:
come on, you can learn this year too:
http://www.tutorialrepublic.com/twitter-bootstrap-tutorial/bootstrap-typography.php